### DISCUSSION TITLE
Searching for documents older than a specified date

### DISCUSSION THREAD

**Discussion by seanrockster** [ravendb/ravendb#19563] [Q&A]
Hi

I've got documents with a datetime property. Is there a way I can return documents older than a specified number of days or compare that datetime property to another datetime value?

Thanks

**Comment by seanrockster** [comment]
I think i've got it, 

session.Advanced.AsyncRawQuery

**Reply by karmeli87** [reply]
Hi,

You would need to create an index for that.
RavenDB can create an [auto-index](https://ravendb.net/docs/article-page/6.2/Csharp/indexes/creating-and-deploying#auto-indexes) for you, by simply running the following query
```
using var session = store.OpenSession();
DateTime targetDate = DateTime.Now.AddDays(-30); // 30 days ago

var results = session.Query<MyDocument>()
                     .Where(doc => doc.CreatedDate < targetDate)
                     .ToList();
```
